nbs/src/core/models.ipynb

```diff
@@ -9951,6 +9960,8 @@
     "                res = {**res, **fitted_pred_int}\n",
     "        if mod['BoxCox_lambda'] is not None:\n",
     "            res_trans = {k : inv_boxcox(v, mod['BoxCox_lambda']) for k, v in res.items()}\n",
+    "            for k, v in res_trans.items():\n",
```

Comment:
I found that in some cases the [inverse Box-Cox ](https://docs.scipy.org/doc/scipy/reference/generated/scipy.special.inv_boxcox.html)generates missing values. One example is series Y11663 from the M4 dataset. This is because there is no x such that $y = (x^\lambda - 1) / \lambda$ for $\lambda \neq 0$. For now, I just returned the non-transformed value.